packages/builder-rsbuild/src/preview/iframe-rsbuild.config.ts

```diff
@@ -288,6 +288,15 @@ export default async (
 
         config.resolve ??= {}
         config.resolve.symlinks = !isPreservingSymlinks()
+        config.resolve.extensions = [
```

Comment:
How about change the default value in Rsbuild?

Reply:
I've considered this option. And I think Storybook has some exclusive convention and may affect the whole Storybook ecosystem. It's better to perfect consistency with Storybook's config to avoid other breaks in the future. Rsbuild's default value could handle most practices, but Storybook is a bit special :P